framework/log/SyslogTarget.php

```diff
@@ -26,6 +26,10 @@ class SyslogTarget extends Target
      * @var int syslog facility.
      */
     public $facility = LOG_USER;
+    /**
+     * @var int openlog option
```

Comment:
documentation should describe the property and list the possible values or refer to the php documentation.
Also `@since 2.0.11` phpdoc annotation is missing.